src/uniprotkb/components/entry/similar-proteins/SimilarProteinsSection.tsx

```diff
@@ -0,0 +1,32 @@
+import { Card } from 'franklin-sites';
+
+import SimilarProteins from './SimilarProteins';
+import LazyComponent from '../../../../shared/components/LazyComponent';
+
+import EntrySection, {
+  getEntrySectionNameAndId,
+} from '../../../types/entrySection';
+
+type Props = {
+  isoforms: { isoforms: string[] };
+  primaryAccession: string;
+};
+const SimilarProteinsSection = ({ isoforms, primaryAccession }: Props) => {
+  const { name, id } = getEntrySectionNameAndId(EntrySection.SimilarProteins);
+  return (
+    <Card
+      header={<h2 data-article-id="similar_proteins_section">{name}</h2>}
+      id={id}
+      data-entry-section
+    >
+      <LazyComponent>
+        <SimilarProteins
+          isoforms={isoforms}
+          primaryAccession={primaryAccession}
+        />
```

Comment:
Since this is lazy-loaded, I guess you can also create a lazy app logic chunk replacing `import SimilarProteins from './SimilarProteins';` so that we can even save loading this logic until the user gets to this section

Reply:
I just did the proposed change in a commit